frontend/pages/playscreen.vue

```diff
@@ -0,0 +1,465 @@
+const currentItem = ref({});
+const timer = ref(null);
+const gameTimer = ref(null); // Neuer Timer für die Spielzeit
+const totalTime = ref(0); // Speichert die Gesamtzeit für den Timer
+const isGameOver = ref(false); // Neuer Zustand zur Überprüfung des Spielendes
+const isGameStarted = ref(false); // Neuer Zustand zur Überprüfung, ob das Spiel gestartet wurde
+const remainingGameTime = ref(60); // Verbleibende Zeit für das Spiel in Sekunden
+const randomizedItems = ref([]); // Zufällig geordnete Items
+
+// Begrenzte Größen für das Dreieck und den Kreis
+const limitedTriangleSize = computed(() =>
+  Math.min(300, Math.max(10, currentItem.value.triangle_size || 10))
+);
+const limitedCircleSize = computed(() =>
+  Math.min(500, Math.max(10, currentItem.value.circle_size || 10))
+);
+
+// Berechnung des Radius und der Größe des Timer-SVGs
+const timerRadius = computed(() => 50); // Feste Größe des Kreises
+const timerSvgSize = computed(() => timerRadius.value * 2 + 10); // SVG-Größe basierend auf dem Radius
+const timerSvgCenter = computed(() => timerRadius.value + 5); // Mittelpunkt des Kreises
+
+// Berechnung für den visuellen Timer-Balken
+const timerDashOffset = computed(() => {
+  const maxDashOffset = 2 * Math.PI * timerRadius.value; // Umfang des Kreises
+  return (remainingTime.value / totalTime.value) * maxDashOffset; // Rückwärtslaufender Balken
+});
+
+// Anzeige des Game Timers als numerische Zeitangabe
+const gameTimerDisplay = computed(() => remainingGameTime.value);
+
+// Funktion zum Mischen des Arrays
+const shuffleArray = (array) => {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+  return array;
+};
+
+// Timer-Logik für das Trisight Mode Spiel
+const startGameTimer = () => {
+  clearInterval(gameTimer.value);
+  gameTimer.value = setInterval(() => {
+    if (remainingGameTime.value > 0) {
+      remainingGameTime.value -= 1;
+    } else {
+      clearInterval(gameTimer.value);
+      endGame(); // Beendet das Spiel, wenn die Zeit abgelaufen ist
+    }
+  }, 1000); // Der Timer reduziert sich jede Sekunde
+};
+
+// Funktion zum Abrufen der Daten basierend auf dem Schwierigkeitsgrad
+const fetchData = async () => {
+  try {
+    let endpoint;
+    if (isTrisightMode.value) {
+      switch (difficulty.value) {
+        case 'medium':
+          endpoint = "http://localhost:8000/api/test_configs/3";
+          break;
+        case 'hard':
+          endpoint = "http://localhost:8000/api/test_configs/4";
+          break;
+        default:
+          endpoint = "http://localhost:8000/api/test_configs/1"; // Leicht (Standard)
+      }
+    } else {
+      endpoint = "http://localhost:8000/api/test_configs/2"; // Endpunkt für den Standardmodus
+    }
+    const response = await $fetch(endpoint);
+    return response;
+  } catch (error) {
+    console.error("Fehler beim Abrufen der Konfigurationen:", error);
```

Comment:
Optimally, we should give feedback to the user because he is not able to play the game in this case but won't understand why. Maybe something like this: https://ui.nuxt.com/components/alert or https://ui.nuxt.com/components/notification